packages/@okta/vuepress-site/docs/guides/email-magic-links-overview/main/index.md

```diff
@@ -0,0 +1,255 @@
+5. Under **EMAIL VERIFICATION EXPERIENCE** enter the URL for the endpoint you created. For example, `http://example.com/magiclink/callback`.
+6. Click **Save** to save your changes.
+
+When required, Identity Engine sends an email similar to this one containing an OTP and a link.
+
+<div class="half border">
+
+![A sample email sent by the Identity Engine with OTP and magic link highlighted](/img/advanced-use-cases/email-magic-links-overview-sample-email.png)
+
+</div>
+
+When a user clicks the magic link from the email authenticator, a request is sent to the Identity Engine to validate the user. It has a token parameter and will look something like this:
+
+`https://${yourOktaDomain}/email/verify/0oa1esny76wdQqjsg697?token=ftlNKCIYA_4WvR6BPbYhyKisVYIF-QLp2t`
+
+Authentication continues at an Okta-hosted page if the EVE setting does not have a value. If the EVE setting has a value, Identity Engine builds a new URL based on the EVE setting and current `otp` and `state` values. Then it redirects the user's browser to that URL. For example
+
+`https://example.com/magiclink/callback?state=1ed1d4de1a50b8f30d87e720bcc1b6a3&otp=439246`
+
+Their authentication process continues there.
+
+<div class="full">
+
+![Flow diagram showing the interaction between application and Identity Engine when using the Verification Experience setting](/img/advanced-use-cases/email-magic-links-overview-experience-flow.png)
+
+</div>
+
+The EVE setting offers a quick way to redirect magic links to your application, but it does require a round trip to the Identity Engine to retrieve the correct URL for the endpoint you have created. Unless you are willing to customize the email templates the email authenticator uses, the user receives only Okta-branded emails. On the other hand, if your user base is multilingual, the default Okta templates support many languages by default. Once you customize an email template, you must create a copy for each language you want to have available.
+
+#### Using custom email templates
+
+Creating custom authentication email templates allows you to send a fully-branded email to your users that skips the round-trip to Identity Engine and redirects users straight to your endpoint.
+
+All Okta email templates are written using [Velocity Templating Language (VTL)](https://help.okta.com/en-us/Content/Topics/Settings/velocity-variables.htm) and use predefined variables to insert relevant values into that email. For example, Okta defines three VTL variables specific to the **Email Challenge** template:
+
+| Variable                     | Contains                                                       |
+|------------------------------|----------------------------------------------------------------|
+| `${verificationToken}`       | The one-time password Okta generated for the user              |
+| `${request.relayState}`      | The current SAML relaystate value                              |
```

Comment:
Is this variable not just for SAML - this is an OAuth parameter IIRC and OIDC protocol doesn't support it directly. Pinging @zeusandrade-okta or @moushmibanerjee-okta to review and clarify this point.

Reply:
I have changed this to " The state token for the current authentication process". I hope that is OK?